StripeConnect/StripeConnect/Source/Models/AccountCollectionOptions.swift

```diff
@@ -20,11 +20,35 @@ public struct AccountCollectionOptions: Equatable, Codable {
         case include
     }
 
+    public enum RequirementsOption: Equatable, Codable {
+        case only([String])
+        case exclude([String])
```

Comment:
It'd be great to add docstrings to these (and all the rest too for that matter - but that's not blocking feedback), that's the standard in our payments SDK e.g. [like this](https://github.com/stripe/stripe-ios/blob/af11b4298a40/StripePayments/StripePayments/Source/API%20Bindings/Models/PaymentMethods/STPPaymentMethod.swift). We often just copy and paste and/or link to the API.

As a user I'd be wondering what values I could pass in here.